src/AWS.elm

```diff
@@ -4,31 +4,89 @@ module AWS exposing (..)
 
 __Experimental: Work in progress__
 
-@docs Credentials, ServiceConfig
+@docs ServiceConfig, Credentials, credentials, AccessKeyId, SecretAccessKey, SessionToken
+
+## Request Signing
+
+@docs Request, SignedRequest, signV4
```

Comment:
Moving most of the package exposed functions and types (other than service stuff) into the root module, to make it simpler to import and use the package